.github/workflows/native-image.yml

```diff
@@ -0,0 +1,108 @@
+name: Build Native Image
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - 'v*'
+
+jobs:
+  native-image:
+    env:
+      LIBSIGNAL_VERSION: 0.17.0
```

Comment:
would be good to have the libsignal version determined automatically, like it's done in https://github.com/AsamK/signal-cli/blob/master/.github/workflows/repackage-native-libs.yml#L40

Reply:
How to do this without downloading the packed version? Can I just read it from a file or gradle property?

Or just using always the latest version?

Reply:
Where does the `libsignal-client-<version>.jar` file come from? Can I download it individually or find it in the repo, so I don't have to download a build every time just to get the version and then build the native image?